src/status_im/common/universal_links.cljs

```diff
@@ -178,13 +178,15 @@
   [{:keys [db]} [{:keys [public-key on-success]}]]
   (let [profile-public-key (get-in db [:profile/profile :public-key])
         profile?           (or (not public-key) (= public-key profile-public-key))
-        ens-name?          (if profile?
-                             (get-in db [:profile/profile :ens-name?])
-                             (get-in db [:contacts/contacts public-key :ens-name]))
+        ;; ens-name?          (if profile?
```

Comment:
@smohamedjavid, what is the intention in keeping code and some unit tests commented in this PR? Would help if we had a comment explaining why we keep the dead code, but even better would be to delete all this code and remove disabled tests and create a GH issue.

Reply:
I just left it in case we bring it back later, but it looks like it will not happen anytime soon (until the ENS code is refactored in status-go). I will remove it as it's a small code update. 👍